Gemfile

```diff
@@ -1,20 +1,17 @@
-ruby '>= 2.7.4'
-
 source "https://rubygems.org"
+ruby '>= 2.7.3'
```

Comment:
Was downgrading to 2.7.3 a requirement?

Reply:
No, meant to change this back - for some reason my local env w/ chruby and ruby-install did not want to use 2.7.4.